README.md

```diff
@@ -19,41 +20,70 @@ Use the provided sql file `nortwhind.sql` in order to populate your database.
  https://docs.docker.com/compose/install/
 
 
-#### 1. run docker-compose
+#### 1. Run docker-compose
 
 ```bash
 > docker-compose up
-
 ...
 ... Lots of messages...
 ...
-Creating network "northwind_psql_default" with the default driver
-Creating northwind_psql_db_1 ... done
-db_1  | 2019-11-28 21:07:14.357 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
-db_1  | 2019-11-28 21:07:14.357 UTC [1] LOG:  listening on IPv6 address "::", port 5432
-db_1  | 2019-11-28 21:07:14.364 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
-db_1  | 2019-11-28 21:07:14.474 UTC [1] LOG:  database system is ready to accept connections
+Creating network "northwind_psql_db" with driver "bridge"
+Creating volume "northwind_psql_db" with default driver
+Creating volume "northwind_psql_pgadmin" with default driver
+Creating pgadmin ... done
+Creating db      ... done
 ```
 
-#### 2. run psql client in the docker-compose container
+#### 2. Run psql client in the docker-compose container
 
 Open another terminal window, and type:
 
 ````bash
 > docker-compose exec db psql -U northwind_user -d northwind
 
-psql (10.5 (Debian 10.5-1.pgdg90+1))
+psql (13.2 (Debian 13.2-1.pgdg100+1))
 Type "help" for help.
 
-postgres=# select * from us_states;
+northwind=# select * from us_states;
  state_id |      state_name      | state_abbr | state_region
 ----------+----------------------+------------+--------------
         1 | Alabama              | AL         | south
         2 | Alaska               | AK         | north
         ...
 ````
 
-#### 3. stop docker-compose
+#### 3. Connect PgAdmin
+
+Access to PgAdmin:
+
+- URL: http://localhost:5050
+- Username: pgadmin4@pgadmin.org (as a default)
+- Password: admin (as a default)
+
+Add new server in PgAdmin:
+
+- Create - Server
+ - Name: db
+ - Host name/address: db
+ - Username: northwind_user
+ - Password: thewindisblowing
+
+- Select DB - northwind
+
+
+#### 4. Connect PyCharm Professional Edition
+
```

Comment:
this works only with Pycharm Professional (I guess), and should be part of a separate PR.